models/Asset/Image/Thumbnail/Processor.php

```diff
@@ -164,7 +164,15 @@ public static function process(Asset $asset, Config $config, mixed $fileSystemPa
             $fileExtension = 'jpg';
         }
 
-        $filename .= '.' . $fileExtension;
+        $configHash = hash("crc32b", serialize([
```

Comment:
Why not use something more modern like [`xxHash`](https://php.watch/versions/8.1/xxHash)?

```suggestion
        $configHash = hash('xxh3', serialize([
```

Reply:
Yeah, why not, but probably `xxh32` to keep the hash shorter.